virttest/utils_libvirt/libvirt_audit.py

```diff
@@ -0,0 +1,38 @@
+"""
+libvirt audit related utility functions
+"""
+
+from avocado.utils import process
+from avocado.utils import service
+
+from virttest import utils_libvirtd
+
+
+def enable_audit_log(libvirtd_config):
+    """
+    Configure audit log level"
+
+    :param libvirtd_config: libvirtd config object
+    """
+    libvirtd_config.audit_level = 1
+    libvirtd_config.audit_logging = 1
+    utils_libvirtd.Libvirtd('virtqemud').restart()
+
+
+def clean_up_audit_log_file():
+    """
+    Clean up audit message in log file.
+    """
+    cmd = "truncate -s 0  /var/log/audit/audit.log*"
+    process.run(cmd, shell=True)
+
+
+def ensure_auditd_started():
```

Comment:
Are these 2 functions related to 'libvirt'? I wonder if it's suitable for us to put auditd service and log related functions udner 'libvirt'. @dzhengfy  What's your opinion?

Reply:
virttest/utils_libvirt/ the path is supposed to include those utilities using vm xml or virsh command.
So from above point, this function could be put into utils_audit.py (for example) under virttest.

Reply:
1)personally I think current implementation for enable_audit_log and clean_up_audit_log_file are very simple because it only satisfy libvirt case's need(more straightforward think, it is just interpreting requirements from libvirt case). It is hard to predict they can satisfy the need other than libvirt test case. So I prefer to keep under utils_libvirt